_appmap/configuration.py

```diff
@@ -219,7 +222,8 @@ def _load_config(self):
                         self._config["name"] = self.default_name
                     if "packages" not in self._config:
                         self._config["packages"] = self.default_packages
-
+                    else:
+                        self._check_path_values()
```

Comment:
Having `_check_path_values` raise an exception is good, because it makes it easy to test.

However, showing the user a stack trace here isn't going to be helpful. Instead, this should catch the exception, show the user the message, and call `sys.exit(1)`.

Reply:
What's the value in continuing in this case? The user has made it clear that they'd like to get recordings (by updating their `appmap.yml`), but the change they've made makes that impossible. If it was me, I'd like to learn that immediately, rather than having the message scroll off my screen as all my tests run....

Reply:
OK